src/main/java/io/ebean/dbmigration/ddlgeneration/platform/PlatformDdl.java

```diff
@@ -390,21 +393,26 @@ public void alterTableAddColumn(DdlBuffer buffer, String tableName, Column colum
     String convertedType = convert(column.getType(), false);
 
     buffer.append("alter table ").append(tableName)
-      .append(" ").append(addColumn).append(" ").append(column.getName())
-      .append(" ").append(convertedType);
+      .append(" ").append(addColumn).append(" ").append(column.getName()).append(" ");
+      
 
     if (!onHistoryTable) {
-      if (isTrue(column.isNotnull())) {
-        buffer.append(" not null");
-      }
+
 
       if (defaultValue != null) {
-        if (typeContainsDefault(convertedType)) {
-          logger.error("Cannot set default value for '" + tableName + "." + column.getName() + "'");
+        int pos = convertedType.indexOf(" default");
```

Comment:
when the convertedType contains already a default value (e.g. 'bit default 0') I remove it and set the correct default value.